lib/cli.js

```diff
@@ -213,6 +214,23 @@ exports.run = async () => { // eslint-disable-line complexity
 
 	const match = arrify(conf.match);
 	const resolveTestsFrom = cli.input.length === 0 ? projectDir : process.cwd();
+
+	const files = cli.input.map(file => path.relative(resolveTestsFrom, path.resolve(process.cwd(), file)));
+	for (const file of files) {
+		try {
+			const stats = fs.statSync(file);
+			if (!stats.isFile()) {
+				exit(`${cli.input[files.indexOf(file)]} should be a file`);
+			}
+		} catch (error) {
+			if (error.code === 'ENOENT') {
+				exit(`No such file: ${cli.input[files.indexOf(file)]}`);
+			} else {
+				exit(`An unexpected error has occurred:\n${error}`);
```

Comment:
Taking the lead from this line:

https://github.com/avajs/ava/blob/2fc7d56475ca1d77078abc3e08390341bedda58f/lib/cli.js#L145

How about:

```js
`Error accessing ${file}\n\n${chalk.gray((error && error.stack) || error)}`
```

Reply:
Yes better